### ISSUE TITLE
Updates to make the repo working with FeathersJs v3

### ISSUE THREAD

**Issue by stratosgear** [lewebsimple/feathers-vuejs#5]
I've forked the repo and tried to make all required changes to make it work with the latest version of FeathersJs v3 (and update versions of other packages) but I have gotten stuck with an "Oh no!
Something went wrong" screen with **no** log messages on what is wrong.

![ohno](https://user-images.githubusercontent.com/193463/36421455-c6de4140-1638-11e8-9d3f-9fe1eeef4da4.png)

Does any one want to have a look if they can spot the error?
https://github.com/stratosgear/feathers-vuejs

Basically, clone, `npm install` and `npm start` to recreate my work!

Thanks!
